Scripts/script-DeleteConext.yml

```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: 8f947b0f-9d53-4f0b-8656-2eb164c02ffa
+  version: 14
+name: DeleteContext
+script: 'return executeCommand(''delContext'', { key: args.key, id: args.id } );'
```

Comment:
add a param - to delete all context keys, and then delete all context keys by iterating on the context ...
Discuss with @nitzanshapira